kratos/solving_strategies/schemes/residual_based_bdf_custom_scheme.h

```diff
@@ -508,25 +542,34 @@ class ResidualBasedBDFCustomScheme
         KRATOS_ERROR_IF(n_variables != n_first_derivative) << "Your list of variables is not the same size as the list of first derivatives variables" << std::endl;
         KRATOS_ERROR_IF(n_variables != n_second_derivative) << "Your list of variables is not the same size as the list of second derivatives variables" << std::endl;
 
-        for (unsigned int i_var = 0; i_var < n_variables; ++i_var){
-            const std::string& variable_name = ThisParameters["variable"].GetArrayItem(i_var).GetString();
-            const std::string& first_derivative_name = ThisParameters["first_derivative"].GetArrayItem(i_var).GetString();
-            const std::string& second_derivative_name = ThisParameters["second_derivative"].GetArrayItem(i_var).GetString();
+        // The current dimension
+        mDomainSize = ThisParameters["domain_size"].GetInt();
```

Comment:
@RiccardoRossi Here

Reply:
ok, however please check that

mDomainSize <= n_variables